src/shared/tpm2-util.c

```diff
@@ -630,9 +646,30 @@ static int tpm2_make_encryption_session(
                         TPMA_SESSION_CONTINUESESSION;
         ESYS_TR session = ESYS_TR_NONE;
         TSS2_RC rc;
-
+        TPM2B_AUTH auth = { 0 };
```

Comment:
we rely on a GNU C extension that allows us to initialize structs to all zero by just writing `= {}`. Hence, please drop the `0`...